src/dsp_tools/commands/excel2json/models/ontology.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+
+@dataclass
+class Property:
+    name: str
+    super: list[str]
+    object: str
+    subject: str | None
+    labels: LanguageDict
+    comments: LanguageDict | None
+    gui_element: str
+    gui_attributes: GuiAttributes | None
+
+    def get(self) -> dict[str, Any]:
+        prop_dict = {"name": self.name, "super": self.super, "object": self.object}
+        if self.subject:
+            prop_dict["subject"] = self.subject
+        prop_dict["labels"] = self.labels.get()
+        if self.comments:
+            prop_dict["comments"] = self.comments.get()
+        prop_dict["gui_element"] = self.gui_element
+        if self.gui_attributes:
+            prop_dict["gui_attributes"] = self.gui_attributes.get()
+        return prop_dict
+
+
+@dataclass
+class LanguageDict:
+    lang_dict: dict[str, str]
+
+    def get(self) -> Any:
```

Comment:
If I did not type with Any then mypy threw an assignment error

Reply:
Then, the underlying problem should be investigated. Simply typing this with Any is not a good solution :-)

Reply:
I have tried everything I can think of, including copilot and chatgpt, they did not suggest anything else that worked. If you could take a look, I would be glad.

Reply:
Done with https://github.com/dasch-swiss/dsp-tools/commit/ea717e12ea5c613a5473a6a9787bea507a87fe4b